setup.cfg

```diff
@@ -70,6 +70,7 @@ test =
     requests
     scipy
     xarray
+    scikit-image
```

Comment:
actually better yet. I'd like to use https://contourpy.readthedocs.io/en/stable/

it doesn't actually add a dependency because matplotlib depends on it.